plugins/deployment_manager/Migration.ts

```diff
@@ -50,16 +48,12 @@ export function getLoader<T>(): Loader<T> {
   return <Loader<T>>loader;
 }
 
-export async function loadMigrations<T>(glob: string): Promise<{ [name: string]: Migration<T> }> {
+export async function loadMigrations<T>(paths: string[]): Promise<{ [name: string]: Migration<T> }> {
   setupLoader<T>();
 
-  const entries = await fg(glob); // Grab all potential migration files
-
-  for (let entry of entries) {
-    let entryPath = path.join(process.cwd(), entry);
-
+  for (let path of paths) {
```

Comment:
Does this change make it harder to run all?

Reply:
Caller can always do `glob`, right? But also I think this is relatively unimportant now, since you never really want to run all, except maybe for deploys, but we'll have 1 (*maybe* 2) of those per deployment, and at worst you would just have to press the button twice (no commit to make in between)